gnomad_bigquery/bq_export_gnomad.py

```diff
@@ -199,7 +186,21 @@ def get_freq_struct(x: hl.expr.ArrayExpression):
     if version == 3 and export_filters:
         ht = ht.annotate(vqsr_filters=vqsr[ht.key].filters)
 
-    export_ht_for_bq(ht, f'{output_dir}/gnomad_v{version}_{data_type}_variants.parquet')
+    if add_vep:
+        vep_ann = vep.select(
+            transcript_consequences=vep.vep.transcript_consequences,
+            most_severe_consequence=vep.vep.most_severe_consequence
+        )
+        ht = ht.annotate(**vep_ann[ht.key])
+
+    if split_variant_table:
+        vep = consequence_filter(vep, least_consequence)
+        v_in_gt_ht = ht.filter(hl.is_defined(vep[ht.key]))
```

Comment:
because here we don't use the AF filter, seems like coding/non-coding would be better then in/not_in gt, no?

Reply:
this applies everywhere in the code

Reply:
Yeah, I went back and forth on this. We dont use the AF filter for GT either so it still applies. My only concern with coding and noncoding was that the 3 prime utr varaint consequence is in the set of noncoding terms. I don't have strong feelings on this though so will change to coding/noncoding.